Resources/Computing/ARCComputingElement.py

```diff
@@ -334,16 +334,25 @@ def getCEStatus(self):
     else:
       # The system which works properly at present for ARC CEs that are configured correctly.
       # But for this we need the VO to be known - ask me (Raja) for the whole story if interested.
-      cmd = 'ldapsearch -x -LLL -H ldap://%s:2135 -b mds-vo-name=resource,o=grid "(GlueVOViewLocalID=%s)"' % (
-          self.ceHost, vo.lower())
-      res = shellCall(0, cmd)
+      # cmd = 'ldapsearch -x -LLL -H ldap://%s:2135 -b mds-vo-name=resource,o=grid "(GlueVOViewLocalID=%s)"' % (
+      #     self.ceHost, vo.lower())
+      if not self.queue:
+        gLogger.error('ARCComputingElement: No queue ...')
+        res = S_ERROR('Unknown queue (%s) failure for site %s' % (self.queue, self.ceHost))
+        return res
+      cmd1 = 'ldapsearch -x -o ldif-wrap=no -LLL -h %s:2135  -b \'o=glue\' "(&(objectClass=GLUE2MappingPolicy)(GLUE2PolicyRule=vo:%s))"' \
+         % (self.ceHost, vo.lower())
+      cmd2 = ' | grep GLUE2MappingPolicyShareForeignKey | grep %s' % (self.queue.split("-")[-1])
+      cmd3 = ' | sed \'s/GLUE2MappingPolicyShareForeignKey: /GLUE2ShareID=/\' | xargs -L1 ldapsearch -x -o ldif-wrap=no -LLL -h %s:2135 -b \'o=glue\' '
```

Comment:
This doesn't work, you need a `\` at the end, or put everything in parenthesis
otherwise the next line gives and `unexpected indent` exception
```python
s1 = 'foo bar ' \
     'baz'

# s3 = 'foo bar'
#      'baz'  # unexpected indent

s2 = ('foo bar '
      'baz')

print 's1',s1
print 's2',s2
```
Also not sure about the order of `%` and implicit string concatenation, maybe more parenthesis

Reply:
seems implicit string concatenation has higher precedence than `%` so this is fine
```python
s1 = 'some'
print ('s1 %s '   's2 ' % s1)  # works
print ('s1 %s ' + 's2 ' % s1)  # fails
```

Reply:
Oh. So should I update my commit? 🤔

Reply:
Hopefully I have addressed your issue - added a \. It seems to run in the LHCb certification instance (so far).